tests/test_record_types.py

```diff
@@ -104,12 +134,16 @@ def test_compare(self):
                 self.assertTrue(metric1 < metric2)
             elif record_type in self.more_is_better_types:
                 self.assertTrue(metric2 < metric1)
+            else:
+                fail
```

Comment:
How about using `self.assertTrue(False, 'An error message')` instead of an undefined variable?